lib/Twig/Extensions/Extension/Date.php

```diff
@@ -97,3 +98,5 @@ public function getName()
         return 'date';
     }
 }
+
+class_alias('Twig_Extensions_Extension_Date', 'Twig\Extensions\Extension_Date', false);
```

Comment:
fixed - ArrayExtension is a workaround the reserved "array" keyword
I'm not sure we should add the extension suffix everywhere (and `Array_` instead of `ArrayExtension` would be worse imho)
PSR-4 fixed in composer.json